openvpn.c

```diff
@@ -167,19 +167,18 @@ void
 OnLogLine(connection_t *c, char *line)
 {
     HWND logWnd = GetDlgItem(c->hwndStatus, ID_EDT_LOG);
-    char *flags, *message;
     time_t timestamp;
     TCHAR *datetime;
     const SETTEXTEX ste = { .flags = ST_SELECTION, .codepage = CP_UTF8 };
 
-    flags = strchr(line, ',') + 1;
-    if (flags - 1 == NULL)
+    char *flags = strchr(line, ',');
+    if (flags == NULL)
     {
         return;
     }
 
-    message = strchr(flags, ',') + 1;
-    if (message - 1 == NULL)
+    char *message = strchr(flags + 1, ',');
```

Comment:
Looks like we got off-by-one error here

`Tue Feb 18 14:26:09 2025 ,Timers: ping 8, ping-restart 40`

Reply:
Fixed